consoleme/lib/aws.py

```diff
@@ -2041,3 +2041,21 @@ async def get_iam_principal_owner(arn: str, aws: Any) -> Optional[str]:
     elif principal_type == "user":
         principal_details = await aws().fetch_iam_user(account_id, arn)
     return principal_details.get("owner")
+
+
+def sanitize_session_name(unsanitized_session_name):
+    """
+    This function sanitizes the session name typically passed in an assume_role call, to verify that it's
+    """
+    import re
+
+    valid_characters_re = r"[\w+=,.@-]"
```

Comment:
To tighten this up a bit, this could be `valid_characters_re = re.compile(r"[\w+=,.@-]")`. Then line 2059 below would change to `if valid_characters_re.match(char):`.

This could be optimized further, but it doesn't seem worth it apart from this low hanging fruit change.

Reply:
Addressed, thanks Patrick